docs(priceId): add an explanation for price id (#116)
dayhaysoos/use-shopping-cart: documentation/src/docs/getting-started.mdx

```jsx
import { useShoppingCart } from 'use-shopping-cart'
import { Product } from './Product'
import { CartItems } from './CartItems'

const productData = [
  {
    name: 'Bananas',
    sku: 'sku_GBJ2Ep8246qeeT',
    price: 400,
    image: 'https://www.fillmurray.com/300/300',
    currency: 'USD'
  },
  {
    name: 'Tangerines',
    sku: 'sku_GBJ2WWfMaGNC2Z',
    price: 100,
    image: 'https://www.fillmurray.com/300/300',
    currency: 'USD'
  }
]

export function App() {
  /* Gets the totalPrice and a method for redirecting to stripe */
  const { totalPrice, redirectToCheckout, cartCount } = useShoppingCart()

  return (
    <div>
      {/* Renders the products */}
      {productData.map((product) => (
        <Product product={product} />
      ))}

      {/* This is where we'll render our cart */}
      <p>Number of Items: {cartCount}</p>
      <p>Total: {totalPrice()}</p>
      <CartItems />

      {/* Redirects the user to Stripe */}
      <button onClick={() => redirectToCheckout()}>Checkout</button>
    </div>
  )
}
```

> Once you have successfully installed `stripe` and `use-shopping-cart` and wrap your root component with CartProvider, you are ready to go! 🚀

Reviewer: ```suggestion
Once you have successfully installed `stripe`, `use-shopping-cart`, and wrapped your root component with `<CartProvider>`, you are ready to go! 🚀
```

### ⚠️ HEADS UP!

> You may not be able to see the SKUs section in your products but a `Prices` section. If that's the case, you can use the price ID (with this shape: `price_abcd1234`) as the sku in yout inventory

Reviewer: ```suggestion
You may not be able to see the SKUs section in your products but a **Prices** section. If that's the case, you can use the price ID (i.e. `price_abcd1234`) as the `sku` property in your inventory.
```
